client/reader/stream/post.jsx

```diff
@@ -24,7 +24,7 @@ class ReaderPostCardAdapter extends React.Component {
 
 	onClick = postToOpen => {
 		let referredPost;
-		if ( get( this.props, 'discoverPick.post' ) ) {
+		if ( get( this.props, 'discoverPost' ) ) {
```

Comment:
this doesn't need to use `get`. Can just be `this.props.discoverPost`.

Reply:
theres a decent amount of cleanup that can still be done all around in terms of `lodash` usage.
We should probably fully get rid of the `PostCardAdapter` and put that functionality inside of `Post`